site/package.json

```diff
@@ -4,6 +4,10 @@
   "private": true,
   "type": "module",
   "dependencies": {
+    "@emotion/react": "^11.14.0",
+    "@emotion/styled": "^11.14.0",
```

Comment:
Was this meant to be used somewhere as a dep or is there a different reason it was installed? I don't see it being imported

Reply:
These are required dependencies for MUI and MUI's impl of material icons. See: https://mui.com/material-ui/material-icons/